[AT-5734] Under maintenance page

okaycj-atat opened the pull request:
Update style for generated Under Maintenance page.  

Also, from the [LGTM docs](https://lgtm.com/help/lgtm/alert-suppression#python), I have supressed the alert for a few lines at the request of reviewers.  


<img width="960" alt="Screen Shot 2020-12-01 at 11 51 50 AM" src="https://user-images.githubusercontent.com/62942635/100770828-b308ad00-33cb-11eb-8d92-7c32c4cc662a.png">
<img width="976" alt="Screen Shot 2020-12-01 at 11 50 07 AM" src="https://user-images.githubusercontent.com/62942635/100770835-b56b0700-33cb-11eb-9a31-350ab3972c71.png">
<img width="960" alt="Screen Shot 2020-12-01 at 11 49 44 AM" src="https://user-images.githubusercontent.com/62942635/100770838-b6039d80-33cb-11eb-80d6-bed3a328c0b1.png">



[Jira](https://ccpo.atlassian.net/browse/AT-5734)

graham-dds reviewed (commented):
Looks great! 

Do you think we could write styles like we do with the rest of the app, rather that in-lining them in the HTML? It would also allow you to write them in a `sass` syntax, which is nice.

You can define those styles as an scss file (maybe in the `sections` directory) and then import it into `atat.scss`. You'll need to `yarn build` to have the styles show up before you generate the page. [Here's](https://gist.github.com/graham-dds/9d5b1bdef78091a78067ede531ecfed2) a diff of what I was thinking.

graham-dds reviewed (approved):
🔨 🔧

graham-dds commented on alembic/env.py:
```diff
@@ -33,7 +33,7 @@
 config.set_main_option("sqlalchemy.url", app_config["DATABASE_URI"])
 
-from atat.database import db
+# from atat.database import db
```
I'm a _little_ scared about commenting this line out, just because it's lived here for so long. Are we sure we don't need to have `db` in scope for some weird reason? It looks like tests pass, but I wonder if this will bite us when we deploy or something. 

If we're sure that we don't need db in scope, can we just delete this line rather than comment it out?

okaycj-atat replied:
Sure thing.  I meant to, it was an error on my part.  Nice catch.

okaycj-atat commented on ipython_setup.py:
```diff
@@ -1,5 +1,4 @@
 from atat.app import make_config, make_app
-from atat.database import db
```
I am sure there is.  I'll make the update.

okaycj-atat commented:
@graham-dds I am not sure what's up with LGMT, but the errors aren't there anymore and I've reverted the code where I removed imports.  Let me know if this is sufficient.

steve-atat reviewed (approved):
👍
